### DISCUSSION TITLE
[Huion H430p] The driver suddenly doesn't work properly today

### DISCUSSION THREAD

**Discussion by RifqiAW** [OpenTabletDriver/OpenTabletDriver#1951] [Q&A] [answered]
I last used my tablet like two weeks ago and it was working properly. Somehow when I tried to use it today it doesn't work. The cursor won't budge but it can click. I tried installing huion official driver and it can finally move again. Then I uninstalled the huion driver and updated my OTD driver and it's still working, however it's not the same as before the whole mess. Idk the exact problem but all I can say is that it felt very different, and I already double checked the tablet area is the same as two weeks ago.

**Comment by jamesbt365** [comment, marked as answer]
Either you have valorant or vanguard running, your settings are invalid (file, reset to defaults then save)

**Reply by RifqiAW** [reply]
I do have valorant and vanguard, so I just need to reset the settings right? Thanks, I will try it after I got home.
